go.mod

```diff
@@ -37,10 +37,10 @@ require (
 	go.etcd.io/bbolt v1.3.11
 	go.etcd.io/gofail v0.2.0
 	go.uber.org/zap v1.10.0
-	golang.org/x/crypto v0.21.0
-	golang.org/x/net v0.23.0
-	golang.org/x/sync v0.5.0
-	golang.org/x/sys v0.18.0
+	golang.org/x/crypto v0.31.0
+	golang.org/x/net v0.33.0
+	golang.org/x/sync v0.10.0
+	golang.org/x/sys v0.28.0
```

Comment:
Bumps from https://github.com/etcd-io/etcd/pull/19154/files
```
	golang.org/x/crypto v0.32.0 // indirect
	golang.org/x/net v0.25.0 // indirect
	golang.org/x/sys v0.29.0 // indirect
	golang.org/x/text v0.21.0 // indirect
```
Any reason to bump `golang.org/x/net` to 33 instead of 25 and `golang.org/x/crypto` to 31 instead of 32?

Reply:
Both net@33 and crypto@31 are the minimal patched versions (can be confirmed by links from PR description)

Reply:
Both work, but I prefer to bumping to the latest version, so 
- golang.org/x/net@v0.34.0
- golang.org/x/net@v0.32.0]

@AwesomePatrol could you update this PR? thx

Reply:
Note 3.5 has already bumped to `golang.org/x/net@v0.34.0` and  `golang.org/x/net@v0.32.0`

Reply:
Done (added in a separate commit)

Reply:
> Done (added in a separate commit)

It makes no sense to have 3 commits for such a small PR. Either you squash it or I will squash it later.

Reply:
Squashed into a single commit